content/blog/2018-11-01-icebreaker.md

```diff
@@ -0,0 +1,90 @@
+
+Please be sure to say both your first and last name regardless of how recognizable some people think you are. Also be aware that people may have different preferred pronouns. For example, when you’re talking about me behind my back 😉, please refer to me as she or her. Someone else might prefer to be referred to as they or them and I hope that people feel comfortable enough here to tell us your preferred pronouns.
+
+Let’s begin…
+
+> Question 1: In the R community there are people who identify as #RDogLadies and #RCatFellas (and #RDogFellas and #RCatLadies). So … if you are a cat lover, move to your right and if you’re a dog person stand over here to your left. People agnostic to these animals, or those who identify as #RChickenLadies, should migrate to the middle.
+
+*Prior to the icebreaker I recruited two people who had extreme different opinions to illustrate how the ice breaker would work. rOpenSci's co-founder [Scott Chamberlain](https://twitter.com/sckottie) is an avowed cat person.*
+
+*Timed 3-minute chat among people standing near each other*
+
+Please introduce yourself and tell us why you have chosen to stand where you are standing...
+
+Does anyone wish to move based on what you’ve heard?
+
+> Question 2: When I need to do something I’ve never done before, I a) consult an outside authority e.g. read the documentation or talk to an expert and try to understand the big picture OR b) I try to find a working example and modify it for my purposes
+
+*Continue with brief introductions to the entire group for 10 to 12 minutes for each question*
+
+> Question 3: I know where I fit as a member of the R community (agree vs disagree)
+
+*Near end of the exercise, ask anyone who did not speak up to introduce themselves to the group.*
+
+## Reflection
+
+> Really work on the questions - they're the key to shifting it from being an ok ice breaker to a really provocative and useful workshop starter. Think about the questions that underpin the workshop, what are the issues people are worried about but unsure how to raise? Can you frame a slightly humorous question that makes it ok to talk about? - Cameron Neylon
+
+The “I know where I fit as a member of the R community” question was ideal coming at the end of the ice breaker because people had become comfortable with the flow of discussion and were willing to be a bit more vulnerable and share slightly more personal opinions. Someone noted that despite having equal representation of women and men in the group there was a much larger proportion of women who indicated “I don't know where I fit in the R community”. One acknowledged expert had themselves standing at the “I don't know where I fit in the R community” end, and they indicated that this was because they suffered from imposter syndrome. This gave me an opportunity to explain [imposter syndrome](https://en.wikipedia.org/wiki/Impostor_syndrome) and subsequently, when we polled the entire group, two thirds said they identified with it!
```

Comment:
```suggestion
The “I know where I fit as a member of the R community” question was ideal coming at the end of the ice breaker because people had become comfortable with the flow of discussion and were willing to be a bit more vulnerable and share slightly more personal opinions. Someone noted that there was a much larger proportion of women in the group who indicated “I don't know where I fit in the R community” than in the other group. One acknowledged expert had themselves standing at the “I don't know where I fit in the R community” end, and they indicated that this was because they suffered from imposter syndrome. This gave me an opportunity to explain [imposter syndrome](https://en.wikipedia.org/wiki/Impostor_syndrome) and subsequently, when we polled the entire group, two thirds said they identified with it!
```

Reply:
The current phrasing sounds a bit binary. You can simply say the proportion of women was higher in one group than the other, after all this is the message no matter what the proportion of women is to begin with.